src/services/jwtService.js

```diff
@@ -0,0 +1,42 @@
+const jwt = require('jsonwebtoken');
+
+require('dotenv').config();
+
+const sign = (user) => {
+  const token = jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: '900s',
+  });
+
+  return token;
+};
+
+const verify = (token) => {
+  try {
+    return jwt.verify(token, process.env.JWT_KEY);
+  } catch (e) {
+    return null;
+  }
+};
+
+const signRefresh = (user) => {
+  const token = jwt.sign(user, process.env.JWT_REFRESH_KEY, {
+    expiresIn: '900s',
```

Comment:
Consider increasing the `expiresIn` duration for the refresh token to a more typical value, such as '7d' or '30d'. The current '900s' (15 minutes) is unusually short for a refresh token and may not provide the intended functionality.